demo.py

```diff
@@ -9,34 +10,39 @@
     "http://citp.princeton.edu/",
 ]
 
+# TODO Write documentation/tutorial on how to import custom browser and manager params
+
 # Loads the default manager params
 # and NUM_BROWSERS copies of the default browser params
-manager_params, browser_params = load_default_params(NUM_BROWSERS)
+manager_params = ManagerParams()
+
+browser_params = []
+for _ in range(NUM_BROWSERS):
+    browser_params.append(BrowserParams(display_mode="headless"))
 
 # Update browser configuration (use this for per-browser settings)
 for i in range(NUM_BROWSERS):
     # Record HTTP Requests and Responses
-    browser_params[i]["http_instrument"] = True
+    browser_params[i].http_instrument = True
     # Record cookie changes
-    browser_params[i]["cookie_instrument"] = True
+    browser_params[i].cookie_instrument = True
     # Record Navigations
-    browser_params[i]["navigation_instrument"] = True
+    browser_params[i].navigation_instrument = True
     # Record JS Web API calls
-    browser_params[i]["js_instrument"] = True
+    browser_params[i].js_instrument = True
     # Record the callstack of all WebRequests made
-    browser_params[i]["callstack_instrument"] = True
+    browser_params[i].callstack_instrument = True
     # Record DNS resolution
-    browser_params[i]["dns_instrument"] = True
-
-
-# Launch only browser 0 headless
-browser_params[0]["display_mode"] = "headless"
```

Comment:
Could you still keep browser 0 in headless mode

Reply:
For sure! Is there a specific reason why it has to be explicitly set so? because I am instantiating BrowserParms something like: `browser_params = BrowserParams(display_mode="headless")`

Reply:
Oh sorry, I missed that. In that case no change is needed. I just got so used to that line being there :sweat_smile: